config/configtelemetry/doc.go

```diff
@@ -4,4 +4,46 @@
 // Package configtelemetry defines various telemetry level for configuration.
 // It enables every component to have access to telemetry level
 // to enable metrics only when necessary.
+//
+// 1. configtelemetry.None
+//
+// No telemetry data should be collected.
+//
+// 2. configtelemetry.Basic
+//
+// Signals associated with this level cover the essential coverage of the component telemetry.
+//
+// This is the default level recommended when running the collector.
+//
+// The signals associated with this level must have low cardinality.
+//
+// The signals associated with this level must represent a small data volume:
+//   - No more than 5 metrics reported.
+//   - At most 1 span actively recording simultaneously, covering the critical path.
+//   - At most 5 log records produced every 30s.
+//
+// Not all signals defined in the component telemetry are active.
+//
+// 3. configtelemetry.Normal
+//
+// Signals associated with this level cover the complete coverage of the component telemetry.
+//
+// The signals associated with this level must control cardinality.
```

Comment:
what does control mean in this context?

Reply:
There is a rationale for the cardinality and the author of the signal can explain the cardinality based on factors.
For example:
* The cardinality is linearly increasing with the number of Kafka topics processed
* The cardinality is correlated to the number of active clients

While this may incur a high cardinality, users can then filter based on it in some way or accumulate the signal by relying on this cardinality.

Reply:
ah I see, thanks for adding that context. I think it would be useful to explain that a bit more in this comment – i.e. "it is acceptable at this level for cardinality to scale linearly with the monitored resources"

Reply:
```suggestion
// The signals associated with this level must control cardinality. It is acceptable at this level for cardinality to scale linearly with the monitored resources.
```